internal/handlers/common/distinct.go

```diff
@@ -115,7 +115,7 @@ func FilterDistinctValues(docs []*types.Document, key string) (*types.Array, err
 				}
 
 				if !distinct.Contains(el) {
-					err := distinct.Append(el)
+					distinct.Append(el)
 					if err != nil {
```

Comment:
I guess that `if` should go away too

Reply:
Yeah, this `if` is not needed.

(Someone should teach me how to do `if err := distinct.Append(el); err != nil {` next time 😅)